docs/source-2.0/guides/smithy-cli/cli_installation.rst

```diff
@@ -0,0 +1,423 @@
+        and then installing the ``smithy-cli`` formula.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+
+            $ brew tap aws/tap && brew install smithy-cli
+
+        After the install completes, you may run ``smithy --help`` to verify your installation.
+
+    .. tab:: Manual (x86)
+
+        First, retrieve the latest smithy installation from the `Smithy GitHub releases`_.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ mkdir -p smithy-install/smithy &&
+                curl -L |release-uri|/|mac-tar| -o smithy-install/|mac-tar| &&
+                tar xvzf smithy-install/|mac-tar| -C smithy-install/smithy
+
+        .. note::
+            If you'd like to verify the downloaded installation, work through :ref:`cli_verification` before proceeding.
+
+        Now, run the installer(``install``) located in the newly created directory(``smithy-install/smithy``).
+
+        .. important:: The following command requires using ``sudo`` to install the CLI in the default
+            location (``/usr/local/smithy``). Alternatively, you may set your own install path, which should
+            mitigate permissions issues when not running with ``sudo``.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+
+            $ sudo smithy-install/smithy/install
+
+        .. note::
+            |install-note|
+
+        After the install completes, you may run ``smithy --help`` to verify your installation.
+        You may also delete all the files downloaded for the installation (``smithy-install/``).
+
+    .. tab:: Manual (ARM)
+
+        First, retrieve the latest smithy installation from the `Smithy GitHub releases`_.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ mkdir -p smithy-install/smithy &&
+                curl -L |release-uri|/|mac-arm-tar| -o |mac-arm-tar| &&
+                tar xvzf smithy-install/|mac-arm-tar| -C smithy-install/smithy
+
+        .. note::
+            If you'd like to verify the downloaded installation, work through :ref:`cli_verification` before proceeding.
+
+        Now, run the installer(``install``) located in the newly created directory(``smithy-install/smithy``).
+
+        .. important:: The following command requires using ``sudo`` to install the CLI in the default
+            location (``/usr/local/smithy``). Alternatively, you may set your own install path, which should
+            mitigate permissions issues when not running with ``sudo``.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+
+            $ sudo smithy-install/smithy/install
+
+        .. note::
+            |install-note|
+
+        After the install completes, you may run ``smithy --help`` to verify your installation.
+        You may also delete all the files downloaded for the installation (``smithy-install/``).
```

Comment:
Same comment here as earlier I think -- I don't know what this is saying. Maybe instead of parens, say "You may also delete all the files downloaded for the installation at `smithy-install/`.